<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSearchCriteria2</class>
 <widget class="QDialog" name="CSearchCriteria2">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>325</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>search criteria</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resfile.qrc">
    <normaloff>:/new/icons/icons/loupe.png</normaloff>:/new/icons/icons/loupe.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="btAdd">
       <property name="text">
        <string>&amp;app</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btIns">
       <property name="text">
        <string>&amp;ins</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btUpd">
       <property name="text">
        <string>&amp;upd</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btDel">
       <property name="text">
        <string>&amp;del</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btOr">
       <property name="text">
        <string>&amp;OR</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btNot">
       <property name="text">
        <string>&amp;NOT</string>
       </property>
       <property name="shortcut">
        <string>Ctrl+S</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="CLatCopt" name="txtInput" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btClear">
       <property name="text">
        <string>&amp;clear all</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="lstWords"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="0" margin="0"/>
 <customwidgets>
  <customwidget>
   <class>CLatCopt</class>
   <extends>QWidget</extends>
   <header>latcopt.h</header>
   <container>1</container>
   <slots>
    <signal>query()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resfile.qrc"/>
 </resources>
 <connections/>
</ui>
